agent/api/container/container.go

```diff
@@ -47,6 +47,13 @@ const (
 
 	// AuthTypeASM is to use image pull auth over AWS Secrets Manager
 	AuthTypeASM = "asm"
+
+	// ContainerMetadataURIEnvironmentVariableName defines the name of the environment
+	// variable in containers' config, which can be used by the containers to access the
+	// v3 metadata endpoint
+	ContainerMetadataURIEnvironmentVariableName = "AWS_CONTAINER_METADATA_URI"
```

Comment:
Is `AWS_CONTAINER_METADATA_URI` intended to be generic across other AWS offerings, or is this an ECS-specific feature?  If it's ECS-specific, I'd recommend renaming this to `ECS_CONTAINER_METADATA_URI`.